src/cmds/restic/cmd_ls.go

```diff
@@ -86,9 +88,20 @@ func runLs(gopts GlobalOptions, args []string) error {
 		return err
 	}
 
-	id, err := restic.FindSnapshot(repo, args[0])
-	if err != nil {
-		return err
+	snapshotIDString := args[0]
+	var id restic.ID
+	var paths []string
+
+	if snapshotIDString == "latest" {
+		id, err = restic.FindLatestSnapshot(repo, paths, "")
```

Comment:
Have a look at how `FindLatestSnapshot` is used in the restore command. There, you can specify the list of paths and a hostname. I think we should add that here, too, to make the user interface consistent.